index.bs

```diff
@@ -163,12 +162,12 @@ When an item is of type <dfn value export for="AV1 Image Item Type">av01</dfn>,
     - The [=AV1 Image Item Data=] shall be identical to the content of an [=AV1 Sample=] marked as 'sync', as defined in [[!AV1-ISOBMFF]].
     - The [=AV1 Image Item Data=] shall have exactly one [=Sequence Header OBU=].
     - If the [=AV1 Image Item Data=] consists of a single frame (i.e. when using a single layer),
-        - It should have its <code>[=still_picture=]</code> flag set to 1.
-        - It should have its <code>[=reduced_still_picture_header=]</code> flag set to 1.
+      - It should have its <code>[=still_picture=]</code> flag set to 1.
```

Comment:
(Lines 24 to 65 specifically were the ones I noticed)

Reply:
I don't have a preference for the indentation level, but it would be good to avoid tabs.

Is there a formatting tool similar to clang-format for Bikeshed files?

Reply:
The PDF versions of HEIF I have render code blocks with an indentation width equivalent to 3 spaces.  
The libavif source files use 4 spaces.  
To keep it inconsistent, I suggest going with 2 spaces for av1-avif.

I will merge this PR as is for now because it contains locally consistent changes and file-wide consistency can be achieved in another PR.